<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameOrchestratorWindow</class>
 <widget class="QMainWindow" name="GameOrchestratorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1388</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VidPokerTerminal</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(6, 139, 3);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
    <item>
     <widget class="QFrame" name="handTableCreditsArea">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="7,3">
       <item>
        <widget class="QFrame" name="handArea">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
          <item>
           <widget class="QFrame" name="multipleHand">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="primaryHand">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4"/>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="payTableArea">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QFrame" name="payoutTable">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="gameLabel">
               <property name="text">
                <string>Poker Game Name</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableWidget" name="payoutDisplay">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="horizontalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QAbstractScrollArea::AdjustToContents</enum>
               </property>
               <property name="showGrid">
                <bool>false</bool>
               </property>
               <property name="columnCount">
                <number>2</number>
               </property>
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>150</number>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <column>
                <property name="text">
                 <string>Hand</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Payout</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="accountBets">
            <property name="styleSheet">
             <string notr="true">font-size: 16pt;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
             <item row="1" column="0">
              <widget class="QLabel" name="betLabel">
               <property name="text">
                <string>Bet</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="creditsLabel">
               <property name="text">
                <string>Credits</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="winLabel">
               <property name="text">
                <string>Win</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="winAmount">
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="betAmount">
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="creditsAmount">
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="softKeyArea">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="returnButton">
         <property name="styleSheet">
          <string notr="true">QPushButton:disabled {
    color: black;
    background-color: gray;
    border-style: outset;
    border-width: 2px;
    border-color: gray;
}
QPushButton {
    color: black;
    background-color: yellow;
    border-style: outset;
    border-width: 2px;
    border-color: gold;
}
QPushButton:pressed {
    color: yellow;
    background-color: black;
    border-style: outset;
    border-width: 2px;
    border-color: yellow;
}
</string>
         </property>
         <property name="text">
          <string>Return</string>
         </property>
         <property name="shortcut">
          <string>N</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="speedButton">
         <property name="styleSheet">
          <string notr="true">QPushButton:disabled {
    color: black;
    background-color: gray;
    border-style: outset;
    border-width: 2px;
    border-color: gray;
}
QPushButton {
    color: black;
    background-color: yellow;
    border-style: outset;
    border-width: 2px;
    border-color: gold;
}
QPushButton:pressed {
    color: yellow;
    background-color: black;
    border-style: outset;
    border-width: 2px;
    border-color: yellow;
}
</string>
         </property>
         <property name="text">
          <string>Speed &gt;&gt;&gt;&gt;</string>
         </property>
         <property name="shortcut">
          <string>M</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="betIncrementButton">
         <property name="styleSheet">
          <string notr="true">QPushButton:disabled {
    color: black;
    background-color: gray;
    border-style: outset;
    border-width: 2px;
    border-color: gray;
}
QPushButton {
    color: black;
    background-color: yellow;
    border-style: outset;
    border-width: 2px;
    border-color: gold;
}
QPushButton:pressed {
    color: yellow;
    background-color: black;
    border-style: outset;
    border-width: 2px;
    border-color: yellow;
}
</string>
         </property>
         <property name="text">
          <string>Increase Bet</string>
         </property>
         <property name="shortcut">
          <string>,</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="maxBetButton">
         <property name="styleSheet">
          <string notr="true">QPushButton:disabled {
    color: black;
    background-color: gray;
    border-style: outset;
    border-width: 2px;
    border-color: gray;
}
QPushButton {
    color: black;
    background-color: yellow;
    border-style: outset;
    border-width: 2px;
    border-color: gold;
}
QPushButton:pressed {
    color: yellow;
    background-color: black;
    border-style: outset;
    border-width: 2px;
    border-color: yellow;
}
</string>
         </property>
         <property name="text">
          <string>Bet Maximum</string>
         </property>
         <property name="shortcut">
          <string>.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="drawDealButton">
         <property name="styleSheet">
          <string notr="true">QPushButton:disabled {
    color: black;
    background-color: gray;
    border-style: outset;
    border-width: 2px;
    border-color: gray;
}
QPushButton {
    color: black;
    background-color: yellow;
    border-style: outset;
    border-width: 2px;
    border-color: gold;
}
QPushButton:pressed {
    color: yellow;
    background-color: black;
    border-style: outset;
    border-width: 2px;
    border-color: yellow;
}
</string>
         </property>
         <property name="text">
          <string>Deal</string>
         </property>
         <property name="shortcut">
          <string>/</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
